cpp/test/Slice/errorDetection/WarningInvalidComment.ice

```diff
@@ -18,7 +18,7 @@ module Test
     }
 
     /// This is a test overview.
-    /// @remark: This is an unknown comment tag which spans 1 line.
```

Comment:
This test is about testing invalid tags.
Now that `@remark` is valid, we needed a new made-up tag instead!